plugins/woocommerce/client/admin/client/settings-payments/components/back-button.scss

```diff
@@ -0,0 +1,6 @@
+.woocommerce-settings-payments__back-button.has-icon {
```

Comment:
Small suggestion about folder structure: what do you think about implementing folders for each component? I saw this pattern used elsewhere in the Core codebase and I think it helps keep our code more organised, especially as we may start also adding tests and assets for a particular component. I have an example in [my PR] (https://github.com/woocommerce/woocommerce/pull/52453/files).

Reply:
Thanks for the suggestion! I don't have strong feelings about this, but it's a good idea for consistency. 👍  Updated in f0c485964e05ade6821e7e927a6b8123719a40fe.

Reply:
Thank you! 🫶